modules/nf-core/fastcov/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "fastcov"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "YOUR-TOOL-HERE"
```

Comment:
please add the tool needed for conda here.

Reply:
The tool is not available on conda sadly :(

Reply:
then a lot of people would be happy if you would add it there 🙂

Reply:
@mashehu I added fastcov as a conda package and included the environment file in the nf-module!